doctrine/event_listeners_subscribers.rst

```diff
@@ -270,3 +270,55 @@ to the tag like so:
 
 .. _`The Event System`: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html
 .. _`the Doctrine Documentation`: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html#entity-listeners
+
+
+Priorities for Event Listeners
+--------------------------------
+
+In case you have multiple listeners for the same event you can control the order in which they are invoked using the `priority` attribute on the tag. Listeners with a higher priority are invoked first.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        services:
+            my.listener.with_high_priority:
+                class: AppBundle\EventListener\MyHighPriorityListener
```

Comment:
This is not the scope of this PR but I don't think we should use the same namespace for kernel listener and doctrine listener (like form listeners should remain in `App\Form\Listener`).